docs/design/hooks-handling.md

```diff
@@ -0,0 +1,62 @@
+# Kata Containers support for `Hooks`
```

Comment:
If you do a new commit for any other reason, I would suggest reordering your commits to put this one first.

Reply:
Update!